docs/contents/releases.md

```diff
@@ -0,0 +1,90 @@
+# Releases
+
+We use the following different release statuses:
+
+* [PROPOSED](#proposed)
+* [PLANNED](#planned)
+* [BUILD](#build)
+* [DEVELOPER PREVIEW](#developer-preview)
+* [BETA](#beta)
+* [GENERALLY AVAILABLE](#generally-available) (GA)
+
+## PROPOSED
+
+The `PROPOSED` status indicates that **the AWS service is on our radar** for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL NOT** be associated with a GitHub Milestone.
+
+## PLANNED
+
+The `PLANNED` status indicates that the AWS service is on our radar for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
+target date for the `DEVELOPER PREVIEW` release of that service's controller.
+
+## BUILD
+
+The `BUILD` status indicates that the ACK service controller for the AWS
+service is **actively being built** in preparation for `DEVELOPER PREVIEW`
+release of that ACK service controller.
+
+## DEVELOPER PREVIEW
+
+The `DEVELOPER PREVIEW` status indicates that the **source code** for ACK
+service controller for the AWS service has been check into the ACK source
+repository along with **minimual end-to-end test cases** that run against a
+local Kubernetes-in-Docker (KinD) cluster.
+
+Notably, an ACK service controller in `DEVELOPER PREVIEW` **DOES NOT include
+Helm charts or published binary Docker images** for easy installation of the
+controller.
+
+The following are release criteria for `DEVELOPER PREVIEW`:
+
+* Source code for the controller checked into ACK source repository
+* Mininal (smoke) tests for at least one resource
```

Comment:
we should say 'service API resources' instead of just 'resources' to be more clear